### PR TITLE
logic: edit usage message in SetCommand

### PR DESCRIPTION
**Pull request by wrewsama**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SetCommand.java [hunk 1]
```diff
@@ -31,14 +36,22 @@
...
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets a person's Contacts. \n"
             + "Parameters: "
+            + PREFIX_NAME + "NAME"
+            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_ROLE + "ROLE "
+            + PREFIX_TIMEZONE + "TIMEZONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_SLACK + "SLACK"
             + PREFIX_TELEGRAM + "TELEGRAM"
```
Comment (Tex-Tang) [inline src/main/java/seedu/address/logic/commands/SetCommand.java:48]: Probably we can add a space at the end to separate different prefixes.

File: src/main/java/seedu/address/logic/commands/SetCommand.java [hunk 2]
```diff
@@ -31,14 +36,22 @@
...
+            + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_ADDRESS + "3755 W Cortaro Farms Rd "
+            + PREFIX_ROLE + "Frontend Engineer "
+            + PREFIX_TIMEZONE + "+8 "
             + PREFIX_EMAIL + "e0123456@u.nus.edu "
             + PREFIX_PHONE + "87654321"
```
Comment (Tex-Tang) [inline src/main/java/seedu/address/logic/commands/SetCommand.java:56]: As above.

**Review by Tex-Tang — CHANGES_REQUESTED** [review]

**Review by Tex-Tang — APPROVED** [review]
Well Done 🥳 !

**Review by sh4nH — APPROVED** [review]
LGTM

**Pull request merged**